$NetBSD$

Better defaults for configuration, inspired by FreeBSD.
\todo Rethink if we should be deviating from upstream.

--- mosquitto.conf.orig	2018-08-16 16:05:26.000000000 +0000
+++ mosquitto.conf
@@ -28,7 +28,7 @@
 # This should be set to /var/run/mosquitto.pid if mosquitto is
 # being run automatically on boot with an init script and 
 # start-stop-daemon or similar.
-#pid_file
+#pid_file /var/run/mosquitto.pid
 
 # When run as root, drop privileges to this user and its primary 
 # group.
@@ -160,7 +160,7 @@
 #bind_address
 
 # Port to use for the default listener.
-#port 1883
+port 1883
 
 # The maximum number of client connections to allow. This is 
 # a per listener setting.
@@ -494,7 +494,7 @@
 # Note that if the broker is running as a Windows service it will default to
 # "log_dest none" and neither stdout nor stderr logging is available.
 # Use "log_dest none" if you wish to disable logging.
-#log_dest stderr
+log_dest syslog
 
 # If using syslog logging (not on Windows), messages will be logged to the
 # "daemon" facility by default. Use the log_facility option to choose which of
